### DISCUSSION TITLE
Timezone is not correct for Asia/Tehran

### DISCUSSION THREAD

**Discussion by amirsarfar** [vercel/next.js#47493] [Help] [answered]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: linux
      Arch: x64
      Version: #1 SMP PREEMPT_DYNAMIC Sun Feb 26 00:13:26 UTC 2023
    Binaries:
      Node: 16.17.1
      npm: 8.15.0
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 13.2.4
      eslint-config-next: 13.2.4
      react: 18.2.0
      react-dom: 18.2.0
```

### Which area(s) of Next.js are affected? (leave empty if unsure)

Internationalization (i18n)

### Link to the code that reproduces this issue

https://github.com/amirsarfar/nextjs-timezone-issue-reproduction-app

### To Reproduce

Just go to index and refresh

### Describe the Bug

![image](https://user-images.githubusercontent.com/36067291/227476148-0f60a53b-10c2-4382-8c6c-384569bfa156.png)

### Expected Behavior

The timezone (Asia/Tehran) was previously a daylight saving timezone (+4:30 in summer and +3:30 in winter) but it has changed this year and is only +3:30.

The server should be updated too and both client and server should be +3:30

**Comment by wwwdepot** [comment, marked as answer]
Nodejs updated its timezone database in v16.19.0. Updating NodeJS to newer version should fix the problem.

https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md

**Reply by amirsarfar** [reply]
Updating to node v16.19.0 solved the problem.

Thanks
